_search-plugins/star-tree-index.md

```diff
@@ -50,15 +50,15 @@ Each node in a star-tree index points to a range of star-tree documents. Nodes c
 
 ### Star nodes
 
-Star nodes are children of non-leaf nodes that contain preaggregated records for data split after dimension removal, aggregating metrics for rows containing dimensions with identical values. These aggregated documents are then appended to the end of star-tree documents. If a document does contain a dimension with identical values, it traverses through the star node.
+Star nodes are nodes which contain the aggregated data of all the other nodes in the same dimension. This helps when querying the aggregated vales of a particular field without traversing through all the nodes in a cluster. 
 
-The star-tree index structure diagram contains the following three examples demonstrating how a document does or does not traverse star-tree nodes (indicated by the `*` symbol in the diagram) during a `Term` query, based on the average request size of the query and whether the document contains matching dimensions:
+The star-tree index structure diagram contains the following three examples demonstrating how a query behaves when retrieving aggregations from nodes in the star-tree, from both the `status` and `port dimensions` and star nodes (indicated by the `*` symbol in the diagram):
 
-- When the port equals `8443` and the status equals `200`. Because the status equals `200`, the query does not traverse through a star node, and the aggregated metric is stored at the end of a star-tree document.
-- When the status equals `200`. The query traverses through a star node in the `port` dimension because `port` is not present as part of the query.
-- When the port equals `5600`. The query traverses through a star node in the `status` dimension because `status` is not present as part of the query.
+- **Blue**: In a `terms` query searching for the computed average request size aggregation, the port equals `8443` and the status equals `200`. Because the query contains values in both the `status` and `port` dimensions, the query returns the aggregation from a non-star node.
+- **Green**: In a `term` query searching for the computed count of requests aggregation, the status equals `200`. Because the query only contains a value from the `status` dimension, the query returns the aggregation from a star node in the `port` dimension.
+- **Red**: In a `term` query searching for the computed average request size aggregation, the port equals `5600`. Because the query does not contain a value from the `status` dimension, the query traverses through a star node in the `status` dimension and returns the aggregation from a star node in the `port` dimension.
```

Comment:
"""
When the port equals `5600`. The query traverses through a star node in the `status` dimension because `status` is not present as part of the query and returns aggregated result from the `5600` child node
"""